### PR TITLE
Continueloginmstests

### PR DESCRIPTION
**Pull request by simondereuver**
Added some basic unit-tests for our inputValidation files. Also scrapped some of the old code and gave up on the idea to mock rabbitmq stuff. Approve if you think it looks good.

### CODE REVIEW

File: package-lock.json
Comment (Pramell94) [inline package-lock.json:1]: Dont want this shit file please

**Review by Pramell94 — CHANGES_REQUESTED** [review]
Looks good just delete that random rackage-lock.json file and we good :)

File: package-lock.json
Comment (simondereuver) [reply package-lock.json:1]: Deleted rackage-lock.json file, pls review agane, ty

**Review by Pramell94 — APPROVED** [review]
lgtm 👍

**Pull request merged**